documentation/src/main/macros/docissimo.py

```diff
@@ -12,13 +12,14 @@
 ```
 attributes:
   versions:
+    camel: 3.13.0
     spec: 2.0.1
     mutiny: 1.1.2
   project-version: '3.14.0-SNAPSHOT'
   smallrye-config-version: '2.7.0'
 ```
 
-You can access values using `{{ attributes.versions.spec}}` or `{{ attributes['project-version'] }}`.
+You can access values using `{{ attributes.versions.camel}}` or `{{ attributes['project-version'] }}`.
```

Comment:
Why changing this one?

Reply:
It is part of the usage docs of this file. I reverse applied the patch we prepared to remove the camel connector docs.

Reply:
Ah funny, didn't remember that!